app/src/androidTest/java/ch/sdp/vibester/profile/ProfileDataProviderTest.kt

```diff
@@ -1,33 +1,33 @@
 package ch.sdp.vibester.profile
 
-import ch.sdp.vibester.profile.ProfileDataProvider
+//import ch.sdp.vibester.profile.ProfileDataProvider
 import ch.sdp.vibester.profile.UserProfile
 import org.junit.Assert
 import org.junit.Test
 
 
 class ProfileDataProviderTest {
 
-    @Test
-    fun checkGetUserData(){
-        val userID = 0
-        val users: List<UserProfile> = listOf(
-            UserProfile("user0", "username0","https://bit.ly/3tO0k8W",  5, 8, 34),
-            UserProfile("user1", "username1", "https://bit.ly/3sZhbXm", 5, 8, 34)
-        )
-        val scoreboard: List<Int> = listOf(1,2,3,4,5,6)
-        val dataProvider = ProfileDataProvider(userID.toString(), users, scoreboard)
-        val finalUser = dataProvider.getUserProfileData()
-
-        // To change on recursive equality of objects
-        Assert.assertEquals(finalUser.handle, users[userID].handle)
-        Assert.assertEquals(finalUser.username, users[userID].username)
-        Assert.assertEquals(finalUser.image, users[userID].image)
-        Assert.assertEquals(finalUser.bestScore, users[userID].bestScore)
-        Assert.assertEquals(finalUser.totalGames, users[userID].totalGames)
-        Assert.assertEquals(finalUser.correctSongs, users[userID].correctSongs)
-        Assert.assertEquals(finalUser.ranking, scoreboard[userID])
-    }
+//    @Test
```

Comment:
If unused and no longer necessary, it would be a good idea to just get rid of these. If you have a use for them later on though, no issues

Reply:
Yeah, as I explained in the comment I commented them out for now before the refactor next week, I'll delete them then. Good catch!